api/api.go

```diff
@@ -64,6 +64,9 @@ func request(ctx context.Context, method string, u url.URL, body io.Reader, head
 	log.Println("[DEBUG] Request URL:", method, req.URL)
 	for k, vs := range req.Header {
 		for _, v := range vs {
+			if k == "Section-Token" {
+				v = "********TOKEN HIDDEN********"
```

Comment:
How did you decide on 8 asterisks on each side? :)

Reply:
I believe the method can be boiled down "LGTM" 😂